SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditor.cpp

```diff
@@ -212,8 +216,30 @@ void FSpatialGDKEditor::GenerateSnapshot(UWorld* World, FString SnapshotFilename
 	}
 }
 
+void FSpatialGDKEditor::WarnIfManualWorkerConnectionSet(FString LaunchConfigPath)
+{
+	FString FileContents;
+	FFileHelper::LoadFileToString(FileContents, LaunchConfigPath.GetCharArray().GetData());
+
+	const FRegexPattern ManualWorkerFlagPattern("\"manual_worker_connection_only\" *: *true");
+	FRegexMatcher ManualWorkerFlagMatcher(ManualWorkerFlagPattern, FileContents);
+
+	if (ManualWorkerFlagMatcher.FindNext())
+	{
+		UE_LOG(LogSpatialGDKEditor, Warning, TEXT("A cloud deployment with \"manual_worker_connection_only\" set to true was launched. This means the server worker will need to be connected manually."));
+		
+		FNotificationInfo Info(FText::FromString(TEXT("Manual worker connection only set for cloud build")));
+		Info.bUseSuccessFailIcons = false;
+		Info.bFireAndForget = false;
+		TSharedPtr<SNotificationItem> NotificationItem = FSlateNotificationManager::Get().AddNotification(Info);
```

Comment:
Is this safe?
I believe notifications must always be done on the game thread / slate thread.